sympy/core/expr.py

```diff
@@ -491,6 +496,18 @@ def is_constant(self, *wrt, **flags):
         True
         """
 
+        def check_denominator_zeros(expression):
+
+            from ..solvers import solveset
+            from ..solvers.solvers import denoms
+
+            free_symbols = expression.free_symbols
+
+            for symb in free_symbols:
+                if any(solveset(d, symb) is not S.EmptySet for d in denoms(expression)):
```

Comment:
I understand you. But I want to calculate the natural domain of the expression. For (1 - x) / (x - 1) the domain of definition I define as the denominator is not zero. So, I would like to solve the equation denominator = 0. Is it denoms(e) zero? I don't now:

`In [1]: from sympy import *

In [2]: x = Symbol('x')

In [3]: e = (1 - x)/(x - 1)

In [4]: d = solvers.denoms(e)

In [5]: for dens in d:
...: print(dens.is_zero)
...:

None
`